### PR TITLE
Added text to credit footer

### PR DESCRIPTION
**Pull request by kbventures**
Added text:  "powered" by ... 

issue: 
https://github.com/whynotearth/foodouken.com/issues/136

### CODE REVIEW

File: src/components/CreditFooter.vue [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <template>
   <div
-    class="flex items-center justify-center text-white align-items-center w-full h-8 text-xs leading-4 primary opacity-38"
+    class="flex items-center justify-center text-white align-items-center w-full h-8 text-xs leading-4 primary"
```
Comment (sally) [inline src/components/CreditFooter.vue:3]: We don't need the classes
* `align-items-center` (doesn't do anything, you already have `items-center`)
* primary (doesn't do anything)

Could we also get some more height on this element? Maybe `h-12`? It looks a bit squished right now compared to the [design file reference](https://user-images.githubusercontent.com/20192983/79815565-dfd4ec00-8335-11ea-9381-1bacf434b243.png)

File: src/components/CreditFooter.vue [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
   >
     <div id="credit">
```
Comment (sally) [inline src/components/CreditFooter.vue:5]: We don't need this `id` attribute on the `div` as we never reference it

File: src/components/CreditFooter.vue [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
-      <span>
-        ⚡by
-        <a class="underline" href="foodouken.com">Foodouken.com</a>
+      ⚡
+      <span class="opacity-38">
+        powered by
+        <a class="underline" href="https://patreon.com/paulchrisluke">Foodouken.com</a>
```
Comment (sally) [inline src/components/CreditFooter.vue:9]: Could we have this link open in a new window? I think it might better for the user. Thoughts @paulchrisluke ?

**Comment by paulchrisluke** [comment]
@kbventures We are finally getting around to getting our licensing squared away, since you've contributed before, would you mind agreeing to our opensource use of your code? https://hub.paulchrisluke.com/paul-chris-luke-agency-opensource-consent-form

**Pull request merged**